content.js

```diff
@@ -113,12 +125,33 @@ function parseGamedataApiXml(str) {
     return game
 }
 
-function findBGGGameId(gameName) {
+function findExactBGGGameId(gameName) {
     query = "https://www.boardgamegeek.com/xmlapi2/search?query=" + String(gameName).replaceAll(' ', '+').replaceAll(':', '+') + "&type=boardgame&exact=1";
     return (
         fetch(query)
             .then(searchResponse => searchResponse.text())
-            .then(searchText => parseSearchApiXml(searchText))
+            .then(searchText => parseExactSearchApiXml(searchText))
+    )
+}
+
+function getGameReleaseYearFromBGA() {
+    let game_info_col = document.getElementsByClassName("col-md-4 game_infos_first_col")[0];
+    if (!game_info_col) return null;
+    let year_row = game_info_col.children[3];
+    if (!year_row) return null;
+    let year_div = year_row.children[1];
+    if (!year_div) return null;
+    let year = year_div.innerHTML;
+    return year;
+}
+
+function findLooseBGGGameId(gameName) {
+    game_release_year_bga = getGameReleaseYearFromBGA();
+    query = "https://www.boardgamegeek.com/xmlapi2/search?query=" + String(gameName.replace(/\(.*\)/i, '')).replaceAll(' ', '+').replaceAll(':', '+') + "&type=boardgame";
```

Comment:
As all these string manipulations are being repeated now, I think it's worth to have it on a specific function.